src/Backend/Core/Js/jquery/jquery.backend.js

```diff
@@ -748,7 +761,7 @@
  */
 (function($)
 {
-	$.fn.tagBox = function(options)
+	$.fn.tagsBox = function(options)
```

Comment:
All i can find is 3rd party plugins.
http://timschlechter.github.io/bootstrap-tagsinput/examples/